README.md

```diff
@@ -1,22 +1,151 @@
-## This project was generated with React.Js & Electron.
+# Tezster GUI
 
-Tezster-GUI comes with pretty much the same functionalities as CLI but in its beta version, it comes integrated with the CLI. After starting tezos node on your local machine with Tezster-CLI, you can clone it from here.
-To start the app in the dev environment run the following command :
+<img  src="https://tezster.s3-ap-southeast-1.amazonaws.com/TEZSTER_GUI/1_jDB5enULQVo2UfeiwD32qA.png"  alt="tezster GUI banner" align="center" />
 
-```bash
-$ npm run dev
-```
+<br />
+
+<div align="center"><strong>Start interacting with Tezos Chain within seconds</strong></div>
+
+<div align="center">Use this GUI version of Tezster which is ready-to-use Desktop app, made with React and Electron. It allows you to interact with Tezos blockchain and to deploy/test smart contracts on Tezos testnets or Sandbox node (which requires <a  href="https://docs.tezster.tech/tezster-cli">Tezster-CLI</a>).
```

Comment:
This is not appropriate description. Previous one had was more detailed. Where's the link for GUI documentation.